apps/dev/dbschema/futures.esdl

```diff
@@ -0,0 +1,4 @@
+# Disable the application of access policies within access policies
+# themselves. This behavior will become the default in EdgeDB 3.0.
+# See: https://www.edgedb.com/docs/reference/ddl/access_policies#nonrecursive
+using future nonrecursive_access_policies;
```

Comment:
I have a slight preference for keeping all the schema information in one file but your call 👍